analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Invocation.cs

```diff
@@ -195,7 +203,6 @@ public static class Extensions
         [DataRow("var dummy = this.Property;")]
         [DataRow("SampleProperty.InstanceMethod();")]
         [DataRow("this.SampleProperty.InstanceMethod();")]
-        [DataRow("this.SampleProperty?.InstanceMethod();")]
         public void Invocation_InstanceMethodCall_DoesNotClearFieldForOtherAccess(string invocation)
         {
             var code = $@"
```

Comment:
Since this method was modified by this PR, it would be good to clean it, by using `"""`. That increases the readability of the content of `code`, since you can properly indent and you don't need `""` on every string delimiter inside `code`.

Reply:
Great, thanks for this change: I find it definitely more readable now!